openapi-v1.yaml

```diff
@@ -4784,6 +4784,27 @@ definitions:
         items:
           $ref: '#/definitions/Enumeration'
 
+  LoadArraySchemaRequest:
+    description: Request to load an array schema
+    type: object
+    properties:
+      config:
+        type: object
+        description: config to use to retrieve the contents
+        $ref: "#/definitions/TileDBConfig"
+
+  LoadArraySchemaResponse:
+    description: Contains the latest schema and all schemas for the opened array
+    type: object
+    properties:
+      latestArraySchema:
```

Comment:
nit: in general we've used snake case for properties

Reply:
Done